### ISSUE TITLE
Chart library using Win2d

### ISSUE THREAD

**Issue by Romasz** [xyzzer/Win2DChartSample#1]
Hello Filip,

thank you for this sample. I'm looking at availble chart projects like [LiveCharts](https://github.com/beto-rodriguez/Live-Charts) or [Telerik](https://github.com/telerik/UI-For-UWP) but seems that none is using DX. Do you know if there is any developed project like this? 

I thought even about cerating new repo for this, but I'm not sure if it's worth to do that - do you have any thoughts on that?

**Comment by VladimirAkopyan** [comment]
I agree, it's a shame that no other charts are GPU accelerated, given the platform's potential to have buffer smooth pretty charts
All kinds of exec types would love that.
We were using Syncfusion charts in our UWP app, and if you just chuck a few hundred thousand data-point into it, it chokes and dies.

**Comment by Romasz** [comment]
@VladimirAkopyan I also see some potential here and I'm looking for solution.

If there is none such project, then I will try to play around a little with basic chart in upcoming days. If I get any results worth to show, I will put them [at GitHub](https://github.com/Romasz/DXCharts).

**Comment by xyzzer** [comment]
Sounds great. It'd be best to think of the contributions model that would make it very easy for people to plug in various renderers to combine different charts with additional visualizations. There are so many different things you might need from a chart that no one person can create them all...

**Comment by Romasz** [comment]
@xyzzer Yeah, this is a huge task. I've got couple of concepts, though I still don't know which one will be the best. I gave a first try to a cartesian chart and put it on GitHub. 

I also wonder if there will be people interested in managing/improving the project.
